### PR TITLE
Update how-to-test-the-prime-api.md

### PR DESCRIPTION
**Pull request by jacquelineIO**
add reference to page using prime-api-client to do acceptance testing

### CODE REVIEW

File: docs/api/testing/how-to-test-the-prime-api.md [hunk 1]
```diff
@@ -144,3 +144,6 @@ prime-api payment-requests POST data.json
 ```
 
 Note that this method for accessing the API is not actively being supported and may be less reliable than others (Postman or the Prime API Client CLI).
+
+## References
+* [Acceptance testing payment requests](/mymove-docs/backend/testing/acceptance-testing-payment-requests/)
```
Comment (jacquelineIO) [inline docs/api/testing/how-to-test-the-prime-api.md:149]:
```suggestion
* [Acceptance testing payment requests](/mymove-docs/backend/testing/acceptance-testing-payment-requests.md)
```

Comment (jacquelineIO) [inline docs/api/testing/how-to-test-the-prime-api.md:149]:
```suggestion
* [Acceptance testing payment requests](/mymove-docs/backend/testing/acceptance-testing-payment-requests)
```

File: docs/api/testing/how-to-test-the-prime-api.md [hunk 2]
```diff
@@ -144,3 +144,6 @@ prime-api payment-requests POST data.json
 ```
 
 Note that this method for accessing the API is not actively being supported and may be less reliable than others (Postman or the Prime API Client CLI).
+
+## References
+* [Acceptance testing payment requests](/mymove-docs/backend/testing/acceptance-testing-payment-requests)
```
Comment (jacquelineIO) [inline docs/api/testing/how-to-test-the-prime-api.md:149]:
```suggestion
* [Acceptance testing payment requests](/mymove-docs/docs/backend/testing/acceptance-testing-payment-requests)
```

File: docs/api/testing/how-to-test-the-prime-api.md [hunk 3]
```diff
@@ -144,3 +144,6 @@ prime-api payment-requests POST data.json
 ```
 
 Note that this method for accessing the API is not actively being supported and may be less reliable than others (Postman or the Prime API Client CLI).
+
+## References
+* [Acceptance testing payment requests](/mymove-docs/docs/backend/testing/acceptance-testing-payment-requests)
```
Comment (jacquelineIO) [inline docs/api/testing/how-to-test-the-prime-api.md:149]:
```suggestion
* [Acceptance testing payment requests](/docs/backend/testing/acceptance-testing-payment-requests)
```

File: docs/api/testing/how-to-test-the-prime-api.md [hunk 4]
```diff
@@ -144,3 +144,6 @@ prime-api payment-requests POST data.json
 ```
 
 Note that this method for accessing the API is not actively being supported and may be less reliable than others (Postman or the Prime API Client CLI).
+
+## References
+* [Acceptance testing payment requests](/docs/backend/testing/acceptance-testing-payment-requests)
```
Comment (jacquelineIO) [inline docs/api/testing/how-to-test-the-prime-api.md:149]: If testing this link from the GH view file page. After clicking not he link you will have to append `.md` to the file for it to show, however in Docusaurus using `.md` will not work.

Comment (rogeruiz) [reply docs/api/testing/how-to-test-the-prime-api.md:149]: Weird. I referenced our internal docs for linking. Which the official Docusaurus documentation says that file based paths work and will be cleaned up automatically. 

=> https://transcom.github.io/mymove-docs/docs/tools/docusaurus/docusaurus#linking-between-pages

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈 this should be good as-is. Let's get this into `main`.

File: docs/api/testing/how-to-test-the-prime-api.md (hunk 4 above)
Comment (jacquelineIO) [reply docs/api/testing/how-to-test-the-prime-api.md:149]: @rogeruiz it works in Docusaurus, it doesn't seem to work in GH (when I tested it).

**Pull request merged**